### PR TITLE
Chore Fixes: Collection Note & Today's Diary

### PR DESCRIPTION
**Pull request by Drapegnik**
depend on https://github.com/babajka/babajka-backend/pull/73

- diary shortcut: https://dev.wir.by/be/diary
- fixed collection note: https://dev.wir.by/en/article/belkino5
- fixed preview: https://dev.wir.by/en/admin/preview/https%253A%252F%252Fwir.fibery.io%252FContent~Marketing%252FArticles-Flow-113%2523Article%252F4-(1970-1974)-41/
- clear utm params: https://github.com/babajka/lib/commit/330f80218bb942f55487bab62e11aed7cfd219e2
- clear `fbclid`: http://dev.wir.by/en/article/belkino5/?fbclid=IwAR2pShqBphqin6tmnZjyruknXKTxWy4b5ZzmhsxQmrm0f_4SKMNEDbVpAbg&utm_source=kek&lol=kek

### CODE REVIEW

**Comment by Drapegnik** [comment]
[fix localization of collection block in articles (UI lang vs Article lang)](https://trello.com/c/9Dfq3Hko/493-fix-localization-of-collection-block-in-articles-ui-lang-vs-article-lang)

**Comment by Drapegnik** [comment]
[to add a redirect from wir.by/diary to today's diary](https://trello.com/c/XLnAe1E9/474-to-add-a-redirect-from-wirby-diary-to-todays-diary)

**Comment by Drapegnik** [comment]
[to remove ?fbid from every url](https://trello.com/c/s9mEOzqk/473-to-remove-fbid-from-every-url)

File: pages/_app.js [hunk 1]
```diff
@@ -108,6 +108,9 @@ class Root extends App {
       ReactGA.initialize(GA_ID, { debug: false });
       ReactGA.ga('send', 'pageview', url, { hitCallback: clearUtmParams });
```
Comment (UladBohdan) [inline pages/_app.js:109]: do we also need to keep `clearUtmParams` here?

**Review by UladBohdan — APPROVED** [review]

File: pages/_app.js (hunk 1 above)
Comment (Drapegnik) [reply pages/_app.js:109]: Yep

**Pull request merged**